### ISSUE TITLE
Updated Linux release for Repositories

### ISSUE THREAD

**Issue by IfGremlinThen** [TASEmulators/desmume#646]
So I ran into an issue in which DeSmuME fails to display the all-important cursor in Advance Wars: Dual Strike.  Look around a bit and you'll find that there is a depth threshold you need to adjust in the "3D Settings", however no such settings exist.

Obviously this indicates that the version you are running may be missing features, and [as one unfortunate user discovered here](https://github.com/TASEmulators/desmume/issues/313) this is because what appears to be **the most popularly distributed Linux version of DeSmuME is, as of now, 7 years old.**

It wasn't obvious to me how dated 9.11 was until I saw the dates on the blogposts on the homepage of the official website, and why would I have 9.11?  Well because...

9.11 is the latest version available in my default Package Manager,
9.11 is the latest version available through [FlatHub.org](https://flathub.org/apps/details/org.desmume.DeSmuME),
9.11 is the latest version available through [Sourceforge.net](https://sourceforge.net/projects/desmume/files/desmume/),
and 9.11 is the latest version advertised on various emulator websites.

Of course if you come here, which redirects you to the blog, *which redirects you to here again,* you'll find a tar.xz file you can download.  But what does that include?  A README.LIN that seems to vaguely suggest I need to download packages from 4 different websites, run terminal commands out of my src directory to mess with my CPU cores, on top of build instructions which, from my experience, are not easily reversible, and it's very unclear how many of these steps are even necessary because I don't know what I should expect to already be on my system, and it's already a gamble whether the developer of any application interprets something as important as a GUI to be an optional step.

I'm a novice Linux user, I've already bricked my machine once just trying to change the color of my window manager and just today I had to abort an installation of an xpad patch because it wanted to screw with my Secure Boot settings, *so of course I'm gunshy at this point*.

It's no wonder that then people Google "desmume deb" only to find the 9.11 version on [pkgs.org](https://pkgs.org/download/desmume) as I did only to find themselves stuck with 7 year old bugs.

I haven't been forced to build *SNES9X, PCSX2, PPSSPP, Visualboy Advance, or Dolphin* from source just to get the game to display properly (and I'd include Mupen64plus in that list if I wasn't experiencing audio issues with it at the moment).

TLDR; Can we please get a new .deb release so the respositories can be updated?

**Comment by zeromus** [comment]
debian packages have metadata which includes "MAINTAINERS" which are the appropriate recipients for your request
e.g. https://packages.debian.org/bullseye/desmume "debian games team"

**Comment by rofl0r** [comment]
just to make something clear: there is no single "linux" we can provide pre-built binaries for, because linux targets dozens of cpu architectures and half a dozen libcs. that's why the instructions explain how to build it from source. but if you happen to use the most popular combination of x86_64 cpu and glibc, chances are that the binaries generated by our CI work on your machine. you can find them on the "actions" tab. e.g. the last build: https://github.com/TASEmulators/desmume/actions/runs/4351272024 (scroll down to artifacts). of course you have to install the libraries it depends on first.

but as zeromus said, if you cant be bothered to build from source, you need to bother your distro's maintainers to provide more recent versions.
